tasks/custom_task/custom_task.py

```diff
@@ -31,6 +32,23 @@ def get_dd_invoke_logs_path() -> str:
     return os.path.join(temp_folder, DD_INVOKE_LOGS_FILE)
 
 
+def get_running_modes() -> dict[str, bool]:
```

Comment:
```suggestion
def get_running_modes() -> list[str]:
```

Reply:
Thanks ! Fixed in [ec3b01d](https://github.com/DataDog/datadog-agent/pull/25860/commits/ec3b01d49b29c9c3e1cf27221dc43921e51da3a4).